docker-compose.yml

```diff
@@ -11,6 +11,8 @@ services:
       - ssl:/etc/cloudify/ssl:ro
       - mgmtworker_workdir:/opt/mgmtworker/work
       - ./resources/rest-service/cloudify/types:/opt/manager/resources/cloudify/types
+    environment:
+      CFY_RESOURCES_ROOT: /tmp/resources
```

Comment:
This really needs to stay the original `/opt/manager/resources/deployments/` (both here and in AIO), because of all the plugins/scripts that already have that path hardcoded

Reply:
```suggestion
      CFY_RESOURCES_ROOT: /opt/manager/resources
```